code/modules/cargo/randomstock.dm

```diff
@@ -1374,9 +1337,6 @@ var/list/global/random_stock_large = list(
 				cr.rigged = 1//Boobytrapped crate, will electrocute when you attempt to open it
 				//Can be disarmed with wirecutters or ignored with insulated gloves
 
-		if ("beesmoker")
```

Comment:
these are actually really fun to fuck around with

Reply:
Buy them from cargo.

Reply:
The beesmoker doesn't come with anything else. It is effectively useless.

Reply:
which also applies to a solid chunk of cargo, except this one is actually fun to toy with

Reply:
I agree with pacman here as well.

Reply:
I'd say keep this as well.